pkg/config/setup/config.go

```diff
@@ -1335,6 +1335,8 @@ func InitConfig(config pkgconfigmodel.Config) {
 
 	// Updater configuration
 	config.BindEnv("updater.registry")
+	config.BindEnv("updater.defaults.catalog_path")
```

Comment:
given that those entries impact what's installed and are sensible, shouldn't they be under a different config owned by dd-updater (spot reserved in updater package creation)

Since they are added only for testing purpose, there's no requirement to be in the config files